Formula/ola.rb

```diff
@@ -1,10 +1,9 @@
 class Ola < Formula
   desc "Open Lighting Architecture for lighting control information"
   homepage "https://www.openlighting.org/ola/"
-  url "https://github.com/OpenLightingProject/ola/releases/download/0.10.7/ola-0.10.7.tar.gz"
-  sha256 "8a65242d95e0622a3553df498e0db323a13e99eeb1accc63a8a2ca8913ab31a0"
+  url "https://github.com/OpenLightingProject/ola/releases/download/0.10.8/ola-0.10.8.tar.gz"
+  sha256 "102aa3114562a2a71dbf7f77d2a0fb9fc47acc35d6248a70b6e831365ca71b13"
   license "GPL-2.0"
-  revision 5
   head "https://github.com/OpenLightingProject/ola.git"
 
   bottle do
```

Comment:
Do the bottle checksums need removing/updating, or does that happen automagically?

Reply:
yeah, this got updated automatically, no need to worry about that.